doc/nrf/doc_styleguide.rst

```diff
@@ -455,20 +225,20 @@ Functions
         * @param[in]  addr        Address to be copied from.
         * @param[in]  len         Number of bytes to copy.
 
-#. If you include more than one **@sa** ("see also", optional), add them this way::
+#. If you include more than one ``@sa`` ("see also", optional), add them this way::
 
       @sa first_function
       @sa second_function
 
-#. **@return** should be used to describe a generic return value without a specific value (for example, "@return The length of ...", "@return The handle").
+#. ``@return`` should be used to describe a generic return value without a specific value (for example, "@return The length of ...", "@return The handle").
    There is usually only one return value.
 
    .. code-block:: none
 
       *  @return  Initializer that sets up the pipe, length, and byte array for
       *           content of the TX data.
 
-#. **@retval** should be used for specific return values (for example, "@retval true", "@retval CONN_ERROR").
+#. ``@retval`` should be used for specific return values (for example, "@retval true", "@retval CONN_ERROR").
    Describe the condition for each of the return values (for example, "If the function completes successfully", "If the connection cannot be established").
```

Comment:
I think these two, and the bullet point following these (lines 481-501/251-271), could be combined to one bullet point how to format the return, with the current ``@return`` and ``@retval`` as sub-bullets for it.